pages/api/self-check/post.ts

```diff
@@ -0,0 +1,17 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { SelfCheckGroup } from "../models/selfCheckGroup";
+
+const postSelfCheck = async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    try {
+        const selfCheck = new SelfCheckGroup({});
```

Comment:
we should get the request body here and populate the selfCheck with it. Currently this would only create an empty object